server/ai_mediaserver.go

```diff
@@ -621,14 +622,144 @@ func (ls *LivepeerServer) StartLiveVideo() http.Handler {
 			GatewayRequestId: &requestID,
 			StreamId:         &streamID,
 		}
-		_, err = processAIRequest(ctx, params, req)
-		if err != nil {
-			stopPipeline(err)
+
+		params.node.LiveMu.Lock()
+		params.node.LivePipelines[params.liveParams.stream] = &core.LivePipeline{
+			RequestID: params.liveParams.requestID,
 		}
-		close(orchSelection)
+		params.node.LiveMu.Unlock()
+		close(pipelineStarted)
```

Comment:
I am not sure if this is correct. The original logic specifically waited until after orchestrator selection (as the variable name indicates).

Right now it will be closing _before_ orchestrator selection which is re-introducing the same bug that https://github.com/livepeer/go-livepeer/pull/3502 fixes (and https://github.com/livepeer/go-livepeer/pull/3483 before it)